client/scripts/controllers/chain/ethereum/token/adapter.ts

```diff
@@ -19,6 +19,7 @@ export default class Token extends IChainAdapter<EthereumCoin, EthereumAccount>
   public readonly class;
   public readonly contractAddress;
   public readonly isToken = true;
+  public readonly isUninitialized: boolean = false;
```

Comment:
`IChainAdapter` already has several different "loaded" or "initialized" members (in addition to what's already on the `IChainModule` as well) -- is there any reason we need a new one?

Reply:
I believe `initialized` is the way to go here too

Reply:
I don't see an `initialized` variable but I'm not sure any of them will do what I need since this is a new behavior relating to how we show these token communities before they have been added to the database. Perhaps I should change the name of the variable it might be confusing, "uncreated" might be better.

Reply:
Token extends IChainAdapter, which is where the `initialized` variable lives.